### ISSUE TITLE
🚸 UX - Distinguishing official planets

### ISSUE THREAD

**Issue by Coca162** [Valour-Software/Valour#1009]
When new users registers for Valour, they have no way of knowing what planets are officially maintained by Valour or not. This has led to confusion between `Valour.Net` (@superjacobl's bot framework) and `Valour Central`.

### Possible solutions
- Bring back automatic joining to the Valour Central planet
- Rank official planets higher in discovery
- Have a tag (Bot, Staff) like system for planets

**Comment by MrLuxuri** [comment]
@Coca162 **Here is another addition to possible solutions as well as an overall improvement suggestion.**

-  At the moment, when you create a new server, users are given the default valour logo, but I suggest replacing that with a + sign and making server icon/pfp selection mandatory/a must instead of optional. This is not only a possible solution for the above issue, but this will also encourage other planets to have their own unique identification, thus making it easier to actually identify not only the official server itself with the official logo but other planets as well.

**Comment by MrLuxuri** [comment]
@Coca162 Yep that would work perfectly as well, I didn't know you folks had a flat design ready, that will def work out.

So for new planets created by other users, give a simple flat version of the logo like the one you attached while the official server will have the real fully-fledged logo.

**Comment by Coca162** [comment]
The issue that caused the need for this has been fixed by changing the default planet icons
